### ISSUE TITLE
Black screen 

### ISSUE THREAD

**Issue by arifzeeshan** [davidtrpcevski/Advanced-LineageOS-Releases#12]
I get a black screen after I clean flash (including Internal Storage) with only ringer slider showing and it bootloops after a while to the same black screen.

LogCat attached. 
[logcat.txt](https://github.com/davidtrpcevski/LineageOS-Releases/files/4357112/logcat.txt)

**Comment by arifzeeshan** [comment]
![IMG_20200320_033339](https://user-images.githubusercontent.com/26674272/77119114-d36e2280-6a5b-11ea-9096-520c36b3252c.jpg)

**Comment by crok** [comment]
Just my two cents:
I'm using [AICP](https://dwnld.aicp-rom.com/) and we had the same issue with the build `aicp_mido_p-14.0-WEEKLY-20200312.zip`
but it's been solved in lates build `aicp_mido_p-14.0-WEEKLY-20200319.zip`.

According to the [changelog of the working build](https://dwnld.aicp-rom.com/device/mido/WEEKLY/aicp_mido_p-14.0-WEEKLY-20200319.zip.html)
the one and only change included in the latest build is this one:
```
project frameworks/base/
d5b58109d2e Add "android.permission.READ_PHONE_STATE" to manifest
```
The change is available [here](https://github.com/AICP/frameworks_base/commit/d5b58109d2ee028fc3f331905ba357f9650080dc).
I hope it will make sense for @davidtrpcevski and it's just something easily implementable.
Fingers crossed.
